src/components/Button/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import { getButtonStyle } from './Button.style';
+
+interface ButtonProps {
+  onClick?: () => void;
+  children: React.ReactNode;
+  variant?: 'filled' | 'outlined'; 
+  disabled?: boolean;
+}

```

Comment:
```suggestion
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  children: React.ReactNode;
  variant?: 'filled' | 'outlined';
}
```
더 유연한 버튼 컴포넌트를 위해 위와 같이 지정해주면 좋을 것 같아요.
해당 코드처럼 확장해주면 onClick, disabled (button에서 기본으로 제공하는) 생략 가능합니다 !

Reply:
onClick, disabled는 생략 가능한 줄 몰랐는데 새로 배웁니다..! 👍👍

Reply:
레전드 리뷰 좋ㄴㅔ요 , , ,